cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -43,7 +43,7 @@ The .NET Framework is the set of APIs that support an advanced type system, data
 #### General cryptography guidance
 
 - **Never, ever write your own cryptographic functions.**
-- Wherever possible, try and avoid writing any cryptographic code at all. Instead try and either use pre-existing secret management solutions or the secret management solution provided by your cloud provider. For more information, see the [OWASP Secrets Management Cheat Sheet](/Secrets_Management_Cheat_Sheet).
+- Wherever possible, try and avoid writing any cryptographic code at all. Instead try and either use pre-existing secret management solutions or the secret management solution provided by your cloud provider. For more information, see the [OWASP Secrets Management Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Secrets_Management_Cheat_Sheet).
```

Comment:
```suggestion
- Wherever possible, try and avoid writing any cryptographic code at all. Instead try and either use pre-existing secret management solutions or the secret management solution provided by your cloud provider. For more information, see the [OWASP Secrets Management Cheat Sheet](/Secrets_Management_Cheat_Sheet.md).
```

Thanks for catching this broken link! The website builder should automatically put in the correct link, but apparently that only works when .md is appended to the URL. See for example the link to the password management cheat sheet later on:

https://github.com/OWASP/CheatSheetSeries/blob/ea12ecba9d281f80859e68b90123bc77adb7cb06/cheatsheets/DotNet_Security_Cheat_Sheet.md?plain=1#L435C1-L435C1

Reply:
Thanks for making me aware, I've added the suggestion.